packages/nextjs/pages/index.tsx

```diff
@@ -129,18 +135,13 @@ const Home: NextPage = () => {
     }
   };
 
-  const fetchAbiFromHeimdall = async (contractAddress: Address) => {
+  const fetchAbiFromHeimdall = async () => {
```

Comment:
We should change this function name, also the catch error message. 

I think we don't need try/catch here so probably we could just inline this function in `onClick` itself it.

Reply:
Done! Thanks!